content/docs/command-reference/params/diff.md

```diff
@@ -19,35 +19,30 @@ positional arguments:
 
 ## Description
 
-This command provides a quick way to compare parameter values among experiments
-in the repository history. Requires that Git is being used to version the
-project params.
+Provides a quick way to compare parameter values among experiments in the
+repository history. Requires that Git is being used to version the project
+params.
 
-> Parameter dependencies are defined with the `-p` option in `dvc run`. See also
-> `dvc params`.
+> Parameter dependencies are defined in the `params` field of `dvc.yaml` (e.g.
+> with the the `-p` (`--params`) option of `dvc run`).
 
 Without arguments, this command compares parameters currently present in the
 <abbr>workspace</abbr> (uncommitted changes) with the latest committed version.
-
-Supported parameter _value_ types are: string, integer, float, and arrays. DVC
-itself does not ascribe any specific meaning for these values.
-
-❗ By default it only shows parameters that were changed.
+This includes everything in `params.yaml` (default parameters file) as well
+everything listed in the `params` field of `dvc.yaml`/`dvc.lock`. Only params
```

Comment:
still don't understand what does `as well everything listed in the `params` field of `dvc.yaml`/`dvc.lock`` mean

Reply:
`params diff` withuot arguments shows everything changed in params.yaml as well as all params defined in dvc.yaml. And soon also even params used to templetize dvc.yaml (but that's not in scope, more for #2052).

I simplified to "...as well all the `params` used in `dvc.yaml`." and separated the mention of dvc.lock into a following sentence.